src/cli/facade/dev.js

```diff
@@ -75,7 +75,14 @@ module.exports = function(dependencies){
         log.warn(strings.messages.cli.PACKAGING_COMPONENTS, true);
 
         async.eachSeries(componentsDirs, function(dir, cb){
-          local.package(dir, false, function(err){
+
+          var packageOptions = {
+            componentPath: dir,
+            minify: false,
```

Comment:
Correct. I changed the signature there, not altering the behaviour. The false there was for minify